Assets/Scripts/DataTypes/GameObjectData.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using UnityEngine;
+
+namespace DataTypes
+{
+    public class GameObjectData<TThis, TBehaviour> : IDisposable
+        where TBehaviour : LinkedBehaviour<TThis>
+        where TThis : GameObjectData<TThis, TBehaviour>
+    {
+        public GameObject gameObject { get; private set; }
+        public TBehaviour behaviour { get; private set; }
+
+        protected void CreateGameObject(GameObject prefab, Vector3 position, Quaternion rotation)
+        {
+            gameObject = UnityEngine.Object.Instantiate(prefab, position, rotation);
```

Comment:
Namespace `UnityEngine` wird bereits bereits verwendet
```suggestion
            gameObject = Object.Instantiate(prefab, position, rotation);
```

Reply:
System hat auch ein Object